games-simulation/singularity/singularity-1.00.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Become a self-aware AI and take over the world"
+DESCRIPTION="Endgame: Singularity is a simulation of a true AI. Go from \
+computer to computer, pursued by the entire world. Keep hidden, and you might \
+have a chance."
+HOMEPAGE="https://singularity.github.io/"
+COPYRIGHT="2005-2023 EMH Software, Singularity community"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitTag="1.00"
+SOURCE_URI="https://github.com/singularity/singularity/archive/v$srcGitTag.tar.gz"
+CHECKSUM_SHA256="5e747268d9e96e69adace1f346fe40d0c3ec05764b98e82cc3ab0335d96d9171"
+SOURCE_DIR="singularity-$srcGitTag"
+ADDITIONAL_FILES="singularity.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+pythonVer="python39"
+
+PROVIDES="
+	singularity$secondaryArchSuffix = $portVersion
+	app:singularity = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:python3.9$secondaryArchSuffix
```

Comment:
```
-	cmd:python3.9$secondaryArchSuffix
+	cmd:python3
```

Reply:
Is this a good idea when the other deps are targeting a specific Python version? This seems like it may break eventually when the `python3` package updates and introduces incompatibilities.

Reply:
There won't be a problem with python packages as they all install in $prefix/lib (despite architecture), in normal recipes we can't mix like this, but this is required because there is no primary package for pillow on 32bit.

Reply:
There is no `$prefix/bin/x86/python3`, the binary is installed in `$prefix/bin`, so no `$secondaryArchSuffix` here and further up.